src/components/aeIdentity/aeIdentity.js

```diff
@@ -67,18 +70,15 @@ export default {
     shortAddress () {
       return this.identity.address.substr(0, 6)
     },
-    classObject: function () {
-      let classes = {
+    chunkAddress () {
+      return this.identity.address.match(/.{1,7}/g)
+    },
+    classObject () {
+      return {
         'ae-identity': true,
         'collapsed': this.collapsed,
-        '_active_yes': this.active,
-        '_active_no': !this.active
+        'active': this.active
       }
-      classes['size_' + this.size] = true
-      return classes
-    },
-    hasSlot () {
```

Comment:
We need this for to access the slot. Why did you remove it?

Reply:
we are not using size anymore, but instead relying on the collapsed attribute.

Reply:
sorry, it was not clear. I was asking about the removal of `hasSlot()` function.